### DISCUSSION TITLE
Other definition

### DISCUSSION THREAD

**Discussion by Darukshock** [SPGoding/datapack-language-server#1125] [General]
Hello ! I have an idea for a potential other sub-command for the `#define`  : `#define storage_path <storage: string>  <path: string> <type: string> [<description: string>]`.
That way, we could be able to manually add an auto-completion for a data path : 
example usage : 
```
#define storage_path namespace:data root.idToChar[].char string the custom font unicode for the item id
data get storage namespace:data root.idToChar[0].
```
The end of line 2 would suggest `char` and show it's a string, (maybe with the description).

**Comment by misode** [comment]
We have some ideas for data pack devs describing the format of storage using mcdoc, which relies on v4 being finished first though.
